generators/app/index.js

```diff
@@ -148,6 +148,12 @@ module.exports = generators.Base.extend({
         },
         {
           type: "confirm",
+          name: "pwa",
+          message: "Would you like to make a Progressive Web App?",
+          default: true
+        },
+        {
+          type: "confirm",
           name: "createDirectory",
           message: "Would you like to create a new directory for your project?",
```

Comment:
this prompt should not appear again when you run `yo electrode` in an existing app.  Use the `when` property.

https://github.com/electrode-io/generator-electrode/blob/master/generators/app/index.js#L102